### ISSUE TITLE
OpenGrid update 311 datasets to reflect SalesForce Service Platform (open data update)

### ISSUE THREAD

**Issue by reve0716** [Chicago/opengrid#334]
311 data has been updated with Salesforce as a service platform.  Our open Data Portal reflect these changes. We need to reflect those same alterations within OpenGrid. Here are the steps to implement this transition.

* [x] 1. Jonathan to configure Open Data Portal data-set.
* [x] 2. Jonathan submit dataset to Plenario 
* [x] 3. Open Grid Team to identify each 311 data-sets that needs to be update in OpenGrid. 311 data-set in OpenGrid is broken into segments as the open data portal.
* [x] 4. Vince (Plenario) to implement the data-set after step 2.
* [x] 5. EKI to configure after step 3.
* [x] 6.  Regina to test the data-sets.

**Comment by reve0716** [comment]
consult with vince how the new items that will be available in future affect vince process on plenario side.

**Comment by reve0716** [comment]
Vince approved Jonathan request for new 311 dataset. Next is to load the data in plenario.

**Comment by reve0716** [comment]
SMS Ticket: 11204

https://chicagogov.sharepoint.com/sites/SMS/_layouts/15/FormServer.aspx?XmlLocation=/sites/SMS/Change/CR_20181211134922.xml&Source=https://chicagogov.sharepoint.com/sites/SMS/SiteAssets/sms/views/sms/closeform.aspx

**Comment by reve0716** [comment]
@rladines  311 saleforce data has uploaded into plenario, need to be configure to OG.

**Comment by rladines** [comment]
@reve0716 Is the expectation to keep all existing 311 broken out datasets and include an all-encompassing one called "311 Service Requests" (maps to plenar.io dataset called _311_service_requests_) like below?

![image](https://user-images.githubusercontent.com/11408107/56045327-acb15300-5d06-11e9-99b1-6ff1aac0b8e1.png)

**Comment by reve0716** [comment]
@rladines in terms of the broken out data-set those would be considered legacy data with an exception to abandoned vehicles which is bringing in current data. To answer your question we can include the 311 Service Requests (311_service_requests) and keep the existing ones since its historical data but we need to identify those data-sets as historical. I test  to identify if all are returning current or data from 2018.

**Comment by rladines** [comment]
@reve0716 How are we planning to distinguish the legacy 311 datasets as 'Legacy'? Can we ask Vince/Plenario to update the human_name attributes of these datasets to say Legacy e.g. "311 Service Requests - Graffiti Removal (_**Legacy**_)"?

**Comment by reve0716** [comment]
@rladines sure i'll send Vince and email to update the human names attributes and cc the team.

**Comment by rladines** [comment]
@reve0716 

The opengrid-plenario-service had been restarted.

This is how the dataset names look now:
![image](https://user-images.githubusercontent.com/11408107/56149653-a58e7d00-5f72-11e9-8ad3-eea17d570020.png)

**Comment by reve0716** [comment]
@rladines I need to contact Vince  about the abandoned vehicle dataset should not be labeled as legacy the dataset is pulling current data but overall it looks great.

**Comment by reve0716** [comment]
@rladines  The abandoned vehicle labeling was resolved.

**Comment by reve0716** [comment]
Created another issue about list of values not being filtered in new dataset see reference issue.

**Comment by reve0716** [comment]
Closing this issue, service request dataset has been added and tested,created offside issue that relates to this dataset setup that needs separate attention.
